.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,80 @@
+name: 🪲 Bug report
+description: Create a report to help us improve
+
+body:
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      options:
+      - label: I have written a descriptive issue title
+        required: true
+      - label: | 
+          I have searched existing issues to ensure the bug has not already been reported
+        required: true
+
+  - type: input
+    id: mongoose-version
+    attributes:
+      label: Mongoose version
+      placeholder: 6.x.x
+    validations:
+      required: true
+
+  - type: input
+    id: node-version
+    attributes:
+      label: Node.js version
+      placeholder: 14.x
+    validations:
+      required: true
+  
+  - type: input
+    id: mongo-version
+    attributes:
+      label: MongoDB version
```

Comment:
To make this more clear, let's put "MongoDB server version".

Many times, people put their version of the [mongodb npm package](https://npmjs.com/package/mongodb), which isn't helpful.